### ISSUE TITLE
Gmail / Outlook Web does not autosuspend

### ISSUE THREAD

**Issue by hongshaoyang** [deanoemcke/thegreatsuspender#705]
Gmail, Outlook Web, and I suspect various other email websites do not auto-suspend because it shows "tab is receiving form input" when TGS is clicked.

The "Do not suspend tabs that contain unsaved form inputs" setting is checked and I do not wish to uncheck it.

### Specifications:
 * Extension version: 6.30
 * Browser & version: Chrome 67.0
 * Operating system & version: macOS Sierra 10.12.6

**Comment by deanoemcke** [comment]
Just to be clear, you're saying that you WANT these tabs to suspend, even though they have form input?
If so, then I assume it's because these webpages are constantly auto-saving user input, and therefore are not affected by tab suspension.
Can you confirm this is indeed what you are requesting?

**Comment by hongshaoyang** [comment]
Yes, I **want** Outlook Web / Gmail to suspend.

![image](https://user-images.githubusercontent.com/19281800/44568853-092d2580-a7ab-11e8-974a-79cdd15d2ac9.png)
For example, when I click the search bar to search for a mail, TGS detects form input. But TGS cannot detect once I am finished with the search. So it does not autosuspend.

**Comment by deanoemcke** [comment]
Yes, it's almost like we need an internal blacklist of websites that have their own auto-saving of form inputs.

Here is a related issue. It suggests 'clearing' the form input status once the form/email is submitted which might help alleviate your issue:
https://github.com/deanoemcke/thegreatsuspender/issues/654

**Comment by liamjohnston** [comment]
How about allowing tabs with form data to be suspended? That's a setting, right? Not ideal, but if the setting is doing more harm than good you could switch it as a sort of workaround for now.

**Comment by liamjohnston** [comment]
How about allowing tabs with form data to be suspended? That's a setting, right? Not ideal, but if the setting is doing more harm than good you could switch it as a sort of workaround for now.

**Comment by hongshaoyang** [comment]
Good point, but I think a better workaround would be to actually fix the logic and behavior of the setting so that it can detect when the form has been suspended, fixing this issue and #654 together.

**Comment by MalikKillian** [comment]
An enhancement that might solve this in some cases is to have a "whitelist" for certain URLs.

When an email is being composed in Gmail the URL ends in something like `/#inbox/FMfgdfgjfgvbnFpNzkmqFhtbGwszSwlKj` but the normal view is just `/#inbox`.  Similarly with Trello, if the URL ends in `/my-board-name` you can't be editing a card.
